.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - "main"
```

Comment:
Can we only maintain one branch? The default branch for GitHub is already `main`.

Reply:
Make sense, thanks